src/interaction/InteractionManager.js

```diff
@@ -675,9 +591,29 @@ export default class InteractionManager extends EventEmitter
         // Resets the flag as set by a stopPropagation call. This flag is usually reset by a user interaction of any kind,
         // but there was a scenario of a display object moving under a static mouse cursor.
         // In this case, mouseover and mouseevents would not pass the flag test in dispatchEvent function
-        this.eventData._reset();
+        for (const k in this.activeInteractionData)
+        {
+            if (Object.prototype.hasOwnProperty.call(this.activeInteractionData, k))
```

Comment:
This would be more efficient, if slightly less safe, if you used `this.activeInteractionData.hasOwnProperty(k)`